apps/prairielearn/src/pages/instructorFileEditor/instructorFileEditor.js

```diff
@@ -340,11 +287,8 @@ async function readEdit(fileEdit) {
     if (draftResult.rows[0].age < 24) {
       fileEdit.editID = draftResult.rows[0].id;
       fileEdit.origHash = draftResult.rows[0].orig_hash;
-      fileEdit.didSave = draftResult.rows[0].did_save;
-      fileEdit.didSync = draftResult.rows[0].did_sync;
```

Comment:
Is it true that the `did_save`/`did_sync` columns of `file_edits` are now unused? If so, let's remove them from the `select_file_edit` query and add a reminder to drop the columns in the future once this has been deployed.

Reply:
I'll do this in another PR as soon as this one is merged and deployed.

Reply:
I opened https://github.com/PrairieLearn/PrairieLearn/issues/9471 so we don't forget about this.